usaspending_api/common/threaded_data_loader.py

```diff
@@ -112,6 +116,17 @@ def load_from_file(self, filepath, encoding='utf-8'):
 
         self.logger.info('Finished processing all rows')
 
+    def csv_file_to_queue(self, csv_file, row_queue):
+        reader = csv.DictReader(csv_file)
+        temp_row_queue = row_queue
```

Comment:
Is this meant to be `temp_row_queue = row_queue.copy()`?

Reply:
Correct, I can change this

Reply:
For reference, Mike and I found out it was more difficult making copies of JoinableQueue (tried .copy and the copy library which didn't work out). The script worked as intended without the copying so we're disregarding this point.